tests/llm/test_ask_holmes.py

```
@@ -98,36 +98,23 @@ def test_ask_holmes(experiment_name, test_case):
-            f"\n** CORRECTNESS **\nscore = {correctness_eval.score}\nrationale = {correctness_eval.metadata.get('rationale', '')}"
-        )
-        scores["correctness"] = correctness_eval.score
-        correctness_span.log(
-            scores={
-                "correctness": correctness_eval.score,
-            },
-            metadata=correctness_eval.metadata,
-        )
+
+    correctness_eval = evaluate_correctness(
+        output=output, expected_elements=expected, parent_span=eval
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)